django-stubs/db/migrations/operations/base.pyi

```diff
@@ -13,6 +14,7 @@ class Operation:
     elidable: bool
     serialization_expand_args: list[str]
     _constructor_args: tuple[Sequence[Any], dict[str, Any]]
+    def __new__(cls, *args: Any, **kwargs: Any) -> Self: ...
```

Comment:
this seems kinda redundant? 🤔 I mean, I guess if stubtest says it's needed...

Reply:
Yep, stubtest complains, seems like the typeshed for object.__new__ is `def __new__(cls) -> Self: ...`